code/datums/actions/order_action.dm

```diff
@@ -52,7 +72,8 @@
 		to_chat(owner, "<span class='warning'>Your last order was too recent.</span>")
 		return
 	TIMER_COOLDOWN_START(owner, COOLDOWN_CIC_ORDERS, ORDER_COOLDOWN)
-	new visual_type(target_turf)
+	if(visual_type)
+		new visual_type(target_turf)
```

Comment:
Why?

Reply:
I've thought of making a new kind of orders without visual types (so it would stick to SL), but finally no